src/main/java/com/zaxxer/nuprocess/windows/NuWinNT.java

```diff
@@ -74,15 +76,15 @@ public HANDLE(Pointer p)
       @Override
       public Object fromNative(Object nativeValue, FromNativeContext context)
       {
-         Object o = super.fromNative(nativeValue, context);
```

Comment:
`super.fromNative` casts `nativeValue` to a `Pointer` and then constructs a new instance using reflection to call the nullary constructor, and then sets the pointer directly on the `pointer` field and returns it.

By doing the cast myself and comparing the pointer directly, I skip creating a full `HANDLE` just to throw it away if it's invalid. This also eliminates the `class.getDeclaredConstructor().newInstance()` reflection and just calls the `HANDLE(Pointer)` constructor directly.

That said, it's probably not super important for performance. `HANDLE` instances _do_ get marshaled from native a lot, though, so it may have some minimal impact.